### PR TITLE
New Random Room: "Fuck you, no random room"

### PR DESCRIPTION
**Pull request by MintyPhresh**
[mapping]
## About the PR
Adds a new 3x3 random room that's filled with reinforced walls. They each say "FUCK YOU!!!" on them, so that when you open the door to the random room you're greeted with a wall that says "fuck you!" like that one door in Fallout.

## Why's this needed?
It's not. Frankly the idea's iffy at best but I got a lot of positive feedback for it on Discord. Might need some rarity changes

### CODE REVIEW

File: assets/maps/random_rooms/3x3/nope_no_room_70.dmm [hunk 1]
```diff
@@ -0,0 +1,28 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/simulated/wall/auto/reinforced,
+/area/dmm_suite/clear_area)
+"G" = (
+/obj/decal/cleanable/writing{
+	words = "FUCK YOU!!!";
+	desc = "Someone's scribbled something here in furious, block-letter text.";
+	uses_default_material_name = s
```
Comment (ZeWaka) [inline assets/maps/random_rooms/3x3/nope_no_room_70.dmm:9]: yeah uhh

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: assets/maps/random_rooms/3x3/nope_no_room_70.dmm (hunk 1 above)
Comment (MintyPhresh) [reply assets/maps/random_rooms/3x3/nope_no_room_70.dmm:9]: Can't tell what this comment is about, or what changes are being requested :/. First time I've had a change req on a mapping PR, still newish

Comment (ZeWaka) [reply assets/maps/random_rooms/3x3/nope_no_room_70.dmm:9]: you were editing the writing and you mistyped `s`

Comment (MintyPhresh) [reply assets/maps/random_rooms/3x3/nope_no_room_70.dmm:9]: Oh, thanks, will fix

**Review by TobleroneSwordfish — APPROVED** [review]
funny, I'd probably make it rarer tho

**Pull request merged**